boom! testing cpython-0.1.0 against stable for nightly
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! e8d366b95ac0f3337703e12ad7658b6641123290569e8f4d3630fa755c163d6d
boom! running `docker start -a e8d366b95ac0f3337703e12ad7658b6641123290569e8f4d3630fa755c163d6d`
kablam!    Compiling cpython v0.1.0 (file:///source)
kablam!    Compiling python3-sys v0.1.3
kablam! error: failed to run custom build command for `python3-sys v0.1.3`
kablam! process didn't exit successfully: `/target/debug/build/python3-sys-93e61270195fe162/build-script-build` (exit code: 101)
kablam! --- stderr
kablam! thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "failed to run python interpreter `\"python\" \"-c\" \"import sys; import sysconfig; print(sys.version_info[0:2]); print(sysconfig.get_config_var(\\\'LIBDIR\\\')); print(sysconfig.get_config_var(\\\'Py_ENABLE_SHARED\\\')); print(sysconfig.get_config_var(\\\'LDVERSION\\\')); print(sys.exec_prefix);\"`: No such file or directory (os error 2)"', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
kablam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
kablam! 
boom! running `docker rm -f e8d366b95ac0f3337703e12ad7658b6641123290569e8f4d3630fa755c163d6d`
blam! e8d366b95ac0f3337703e12ad7658b6641123290569e8f4d3630fa755c163d6d
